### PR TITLE
fix(core): missed layer to layer.source 

### PR DESCRIPTION
**Pull request by zarov**
Original message: Few things I did in #849 that can be done outside #849.

### CODE REVIEW

File: src/Core/Scheduler/Scheduler.js [hunk 1]
```diff
@@ -160,7 +160,7 @@ Scheduler.prototype.execute = function execute(command) {
 
     // parse host
     const layer = command.layer;
-    const host = layer.url ? new URL(layer.url, document.location).host : undefined;
+    const host = layer.source && layer.source.url ? new URL(layer.source.url, document.location).host : undefined;
```
Comment (gchoqueux) [inline src/Core/Scheduler/Scheduler.js:163]: it doesn't work for 3Dtiles by examples?

**Comment by gchoqueux** [comment]
@zarov Could you split in two PRs to merge OBB

**Comment by zarov** [comment]
@gchoqueux done, with #871

**Comment by zarov** [comment]
Rebased !

File: src/Core/Scheduler/Scheduler.js (hunk 1 above)
Comment (gchoqueux) [reply src/Core/Scheduler/Scheduler.js:163]: What is the behavior with `3dTiles` and `point cloud`?

Comment (zarov) [reply src/Core/Scheduler/Scheduler.js:163]: I see no problem in the 3dtiles example, is this what you meant ?

Comment (gchoqueux) [reply src/Core/Scheduler/Scheduler.js:163]: `3dtiles` haven't `Source`, so `host` is `undefined`

Comment (zarov) [reply src/Core/Scheduler/Scheduler.js:163]: `host` being `undefined` doesn't seem to make any problem, other than it is not in the `hostQueues`. The URL here for `3dtiles` corresponds to the root file of 3dtiles, which is fetch inside the provider in `preprocessDataLayer`. So no need to add it in the `hostQueues` as far as I know. We can change this behavior when rewriting the 3dtiles system.

**Comment by gchoqueux** [comment]
Could you rebase this branch?

**Comment by zarov** [comment]
Done

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**